sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -793,6 +781,16 @@
                   <release>11</release>
                 </configuration>
               </execution>
+              <execution>
+                <id>default-testCompile</id>
+                <goals>
+                  <goal>testCompile</goal>
+                </goals>
+                <configuration>
+                  <release>11</release>
+                  <testRelease>11</testRelease>
```

Comment:
Needed to override the `default-testCompile` to use Java 11 in the build instead of Java 17